### PR TITLE
Attempt to close #146

### LINKED ISSUE #146: fct_count(..., prop_OR_relative_OR_normalize_OR_freq= T)
I wasn't able to find a good short approach to do
```
c("a", "a", "b") %>% table() %>% prop.table()
```
using ```forcats``` package. I think it's pretty handy to use a special argument for this purpose as in [sklearn value_counts](https://pandas.pydata.org/pandas-docs/version/0.17.0/generated/pandas.Series.value_counts.html)

### PR DESCRIPTION
**Pull request by zhiiiyang**

### CODE REVIEW

File: R/count.R [hunk 1]
```diff
@@ -3,14 +3,16 @@
 #' @param f A factor (or character vector).
 #' @param sort If `TRUE`, sort the result so that the most common values
 #'   float to the top.
-#' @return A tibble with columns `f` and `n`.
+#' @param prop If `TRUE`, compute the fraction of marginal table
+#' @return A tibble with columns `f`, `n` and `p`.
```
Comment (hadley) [inline R/count.R]: and `p`, if `prop` is `TRUE` 

?

Comment (zhiiiyang) [reply R/count.R]: Yes. You are right! Thanks.

**Comment by hadley** [comment]
It looks like you've accidentally kept working in the same branch. This is ok for today, but it would also be a good time to practice disentangling if you wanted some in person support.

**Comment by zhiiiyang** [comment]
I realize it just now and that is why I undid the latter commits. Sorry about that.

**Comment by hadley** [comment]
Ok great :)

**Comment by hadley** [comment]
Can you please add a bullet to NEWS? It should briefly [describe the change](http://style.tidyverse.org/news.html) and end with `(@yourname, #issuenumber)`.

**Comment by hadley** [comment]
Thanks!!

For future reference, it's helpful if you make the PR title a brief summary ("adds `prop` argument to `fct_count()`") and includes `Fixes #146` in the description.

**Pull request merged**